src/script/page/MainContent/panels/preferences/DevicesPreferences/components/E2EICertificateDetails/E2EICertificateDetails.tsx

```diff
@@ -73,7 +73,7 @@ export const E2EICertificateDetails = ({identity, isCurrentDevice}: E2EICertific
             onClick={() => setIsCertificateDetailsModalOpen(true)}
             data-uie-name="show-certificate-details"
             aria-label={t('E2EI.showCertificateDetails')}
-            tabIndex={TabIndex.FOCUSABLE}
+            //tabIndex={TabIndex.FOCUSABLE}
```

Comment:
question: Is that intentional? If so, let's just remove it

Reply:
nice find